.bintray/.credentials

```diff
@@ -0,0 +1,4 @@
+realm = RasterFoundry API Realm
```

Comment:
I'd say that leaving these with the Bintray defaults is potentially better because using RF specific endpoints might mislead people:

```
realm = Bintray API Realm
host = api.bintray.com
user =
password =
```